src/extension.ts

```diff
@@ -156,13 +158,14 @@ export function deactivate(): void {
 }
 
 async function registerOpenShiftLocalCluster(
+  name,
   provider: extensionApi.Provider,
   extensionContext: extensionApi.ExtensionContext,
 ): Promise<void> {
   const status = () => crcStatus.getConnectionStatus();
   const apiURL = 'https://api.crc.testing:6443';
   const kubernetesProviderConnection: extensionApi.KubernetesProviderConnection = {
-    name: 'OpenShift',
+    name,
```

Comment:
So this now is recognized as 'OpenShift Local', right?

Reply:
Yes https://github.com/crc-org/crc-extension/pull/48/files#diff-04bba6a35cad1c794cbbe677678a51de13441b7a6ee8592b7b50be1f05c6f626R222